python/ray/train/torch/config.py

```diff
@@ -104,6 +105,9 @@ def _setup_torch_process_group(
             "To override this behavior, you can set NCCL_ASYNC_ERROR_HANDLING=0."
         )
         os.environ["NCCL_ASYNC_ERROR_HANDLING"] = "1"
+    elif HPU_PACKAGE_AVAILABLE and backend == "hccl":
```

Comment:
nit for readability: 

```suggestion
    elif backend == "hccl" and HPU_PACKAGE_AVAILABLE:
```